notebooks/official/experiments/comparing_local_trained_models.ipynb

```diff
@@ -32,51 +32,48 @@
         "<table align=\"left\">\n",
```

Comment:
<p>Use the current template version for PROJECT_ID</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/971/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
<p>done</p>